packages/app/src/elements/item-view.ts

```diff
@@ -654,6 +687,41 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
                             </div>
                         </div>
 
+                        <div class="history">
+                            <h2
+                                class="subtle horizontally-double-margined bottom-margined animated section-header"
+                                style="margin-left: 1.2em;"
+                            >
+                                <pl-icon icon="history" class="inline small light"></pl-icon>
+                                ${$l("History")}
+                            </h2>
+
+                            <pl-list class="border-top block" ?hidden=${!history.length}>
+                                ${history.map((historyEntry) => {
+                                    const historyEntryUpdater =
+                                        org && org.getMember({ accountId: historyEntry.updatedBy });
+
+                                    return html`
+                                        <div
+                                            class="double-padded small hover click border-bottom horizontal center-aligning text-centering"
+                                            @click=${() => this._showHistoryEntry(historyEntry)}
+                                        >
+                                            ${$l("Changed")} ${historyEntry.fieldsChanged.join(", ")} ${" "}
+                                            ${until(formatDateFromNow(historyEntry.date))}
+                                            ${historyEntryUpdater && " " + $l("by {0}", historyEntryUpdater.email)}
```

Comment:
Could you elaborate? Show the actual value changes here? I was trying to prevent that because of potentially showing passwords.

Reply:
As explained in another comment:

> Again, I'd like the whole thing to be a list of snapshots, rather than a list of changes. I.e.: "What did the item look like at a certain point in time" vs "What has changed and when". Remember the main point of this feature is not to serve as an audit log but simply to allow people to restore a previous version of an item.

I **don't** want to display a list of changes (or at least not focus on them) but instead simply show that a different version of this item existed at a certain point in time. They can then "travel back" to that time and see how the item looked back then and, if desired, restore that point in time. Very similar to how time machine works on MacOS.